docs/config.md

```diff
@@ -12,6 +12,7 @@ Here's a quick overview of what everything means with an example file:
 ```php
 <?php return array (
   'debug' => false, // enables or disables debug mode, used to troubleshoot issues
+  'offline' => false, // enables or disables site maintenance mode
```

Comment:
```suggestion
  'offline' => false, // enables or disables site maintenance mode. This makes your site inaccessible to all users (including admins).
```

Thanks, this is somewhat of an "unknown" feature. I think it would be good to clarify what this does, so people don't get confused (at least until https://github.com/flarum/core/issues/1732 gets fixed).